src/custom/pages/Faq/index.tsx

```diff
@@ -173,6 +173,15 @@ export default function Faq() {
             features while placing gas free trades.
           </p>
 
+          <h3 id="why-is-cowswap-a-meta-dex-aggregator">Why is CowSwap a Meta DEX aggregator?</h3>
+          <p>
+            Cowswap is built on top of the Gnosis protocol which matches trades via batch auctions on a variety of on-chain liquidity.
+            Trades can be settled via underlying on-chain AMMs directly or via DEX Aggregators, depending on which pool /path offers 
+            the best price. It is thus essentially acting as a DexAggregator of the DexAggregators.  In addition to that, before 
+            finding the best trading price for a trade from the available on-chain liquidity, Gnosis Protocol first checks if it 
+            can find a coincidence of wants within the existing batch to offer an even better price than any pool can.
```

Comment:
This is an example where I want to make a suggested change for the sentence but have to suggest over multiple lines.

```suggestion
            finding the best price for a trade from available on-chain liquidity, Gnosis Protocol first seeks a coincidence of wants within the existing batch to offer an even better price than any pool can.
```